custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -103,6 +124,20 @@ def input_values(cls) -> List["Values"]:
     def output_values(cls) -> List["Values"]:
         return [cls.NEVER, cls.DYNAMIC, cls.ALWAYS, cls.IDENTITY]
 
+    @classmethod
```

Comment:
same comment as above, please move to the tests which are the only place they're used